OfficeUIFabric.Demo/src/main/java/com/microsoft/officeuifabricdemo/demos/AvatarViewActivity.kt

```diff
@@ -37,6 +38,9 @@ class AvatarViewActivity : DemoActivity() {
         avatar_example_large_initials_square.name = getString(R.string.persona_email_henry_brill)
         avatar_example_large_initials_square.avatarStyle = AvatarStyle.SQUARE
 
+        // Change initials background color
+        avatar_example_xsmall_initials.backgroundColor = Color.RED
```

Comment:
Not that the xsmall avatar is easy to ready anyway... But could you make this one a bit darker for accessibility too?  Maybe `DKGRAY` or something custom?